tests/unit/services/cookies-test.js

```diff
@@ -267,39 +280,39 @@ describeModule('service:cookies', 'CookiesService', {}, function() {
       });
 
       it('returns undefined for a cookie that was written for another path', function() {
-        this.fakeFastBoot.request.path = '/path';
+        this.subject().set('_fastBoot.request.path', '/path');
```

Comment:
This shouldn't make a difference and the first one is a bit more concise actually.